SConstruct

```diff
@@ -743,10 +743,12 @@ if 'all' in download_dependencies:
 # dependencies that should be pre-installed on system
 system_dependencies = all_dependencies - download_dependencies
 
+env.FindPkgConfig(toolchain)
```

Comment:
Let's move this to the end of this `conf` block: https://github.com/roc-project/roc/blob/38cd0704108edb73263ef504462c50995e6a32e9/SConstruct#L570

Note that this call should be before this block:

https://github.com/roc-project/roc/blob/38cd0704108edb73263ef504462c50995e6a32e9/SConstruct#L694-L699

otherwise, cloned envs wont have correct PKG_CONFIG var.